app/serializers/person_escort_record_serializer.rb

```diff
@@ -0,0 +1,25 @@
+class PersonEscortRecordSerializer < ActiveModel::Serializer
+  belongs_to :profile, record_type: :profile
+  belongs_to :framework, record_type: :framework
+  has_many :framework_responses, serializer: FrameworkResponseSerializer, key: :responses
+
+  attributes :version, :status
+
+  def status
+    object.state
```

Comment:
Why is there a status and a state attribute, or is this just remapping things to the API view of the model?

If this is intentional, then perhaps use a more idiomatic AMS approach:

```
attribute :status, key: :state
```

Reply:
Yea it's just remapping things for the API view, I liked keeping state on the model. I'll amend to use key instead.